tests/test-projection.cc

```diff
@@ -147,6 +147,62 @@ BOOST_AUTO_TEST_CASE (projectToComPosition) {
   }
 }
 
+/*
+BOOST_AUTO_TEST_CASE (projectToComPositionSimpleHumanoid) {
 
+  RbPrmFullBodyPtr_t fullBody = loadSimpleHumanoid();
+  const std::string rLegId("rleg");
```

Comment:
Why is this commented ?

Reply:
Sorry I mean, should we wait rather than commit a commented test ?

Reply:
I didn't  remove anything in this file, but I tried to add a new test case with another robot model. This new test case is not working right now but i need a version with all the tests working in order to finish the continuous integration setup as soon as possible. That's why I commited a work in progress / commented  code.

This test still cover exactly the same things as before.